EasyPost/Models/Shared/Collection.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using EasyPost._base;
+using Newtonsoft.Json;
+
+namespace EasyPost.Models.Shared
+{
+    public class Collection : EasyPostObject
+    {
+        #region JSON Properties
+
+        [JsonProperty("has_more")]
+        public bool has_more { get; set; }
+
+        [JsonProperty("filters")]
+        internal Dictionary<string, object>? filters { get; set; }
+
+        #endregion
+
+        protected void UpdateFilters(IEnumerable<EasyPostObject>? elements, string propertyName)
```

Comment:
Question: Can you walk me through how this works? This could benefit from a docstring. Can users pass `after_id` and `before_id`? What about `page_size`?

Reply:
Going to remove this for now, we can address collections in the future.